components/player-loader.tsx

```diff
@@ -0,0 +1,69 @@
+import { forwardRef } from 'react';
+import { breakpoints } from '../style-vars';
+import { HTMLVideoElementWithPlyr } from '../types';
+import dynamic from 'next/dynamic';
+
+const PlyrPlayer = dynamic(() => import('./plyr-player'));
+
+type Props = {
+  playbackId: string
+  poster: string
+  currentTime?: number
+  aspectRatio: number;
+  onLoaded: () => void;
+  playerType: string;
+  onError: (error: ErrorEvent) => void;
+};
+
+const PlayerLoader = forwardRef<HTMLVideoElementWithPlyr, Props>(({ playbackId, poster, currentTime, aspectRatio, playerType, onLoaded, onError }, ref) => {
+  const isVertical = aspectRatio < 1;
+
+  return (
+    <>
+      <div className='video-container'>
```

Comment:
TODO:

based on the `aspectRatio` (can also pass down `width` and `height` if we need it) we should render this `video-container` in the right dimensions to prevent a layout shift happening when the player dynamically loads)